clmm/modbackend/numcosmo.py

```diff
@@ -228,7 +228,7 @@ def _get_param(self, key):
 
     def set_dist(self, dist):
 
-        assert isinstance(dist, Nc.Distance)
+        #assert isinstance(dist, Nc.Distance)
```

Comment:
Just curious why this is not needed anymore...

Reply:
this change should not have been committed. Good thing you caught it :wink: